src/main/java/com/openshift/internal/client/ApplicationResource.java

```diff
@@ -1036,7 +1036,7 @@ protected InputStream getInputStream(Channel channel) throws IOException {
 		public List<String> getLines(Channel channel) throws IOException {
 			BufferedReader reader = new BufferedReader(new InputStreamReader(getInputStream(channel)));
 			List<String> lines = new ArrayList<String>();
-			String line = null;
+			String line;
```

Comment:
-1, method variables are not initialized implicitily (unlike instance variables). It is therefore always preferable to initialize them.